packages/@react-spectrum/list/src/ListViewItem.tsx

```diff
@@ -53,14 +56,22 @@ export function ListViewItem(props) {
     node: cellNode,
     focusMode: 'cell'
   }, state, cellRef);
```

Comment:
We should add `shouldSelectOnPressUp: true` to `useGridRow` if DnD is enabled so that selection isn't toggled on mouse down when you meant to drag.

Reply:
I noticed that with `shouldSelectOnPressUp: true`, clicking the row's checkbox won't select the item. I'll see if this is a small enough fix to just include in here.